src/jit/inlinepolicy.h

```diff
@@ -216,13 +216,45 @@ class DiscretionaryPolicy : public LegacyPolicy
     const char* GetName() const override { return "DiscretionaryPolicy"; }
 
 private:
+    
+    void ComputeOpcodeBin(OPCODE opcode);
+    enum { MAX_ARGS = 4 };
```

Comment:
I have the full data set lying around... 4 is 1.6%, 5 is 0.29%, 6 is 0.19%. So I suppose increasing to include data from 4 arg inlinees would make sense.

So far, none of this argument information plays a significant role in the code size models I've developed. This is somewhat counter-intuitive. Expect some further refinement here once I figure out what is going on.